README.md

```diff
@@ -72,6 +73,184 @@ $myEntity = Stub::extends($myEntity, ['property1' => 'value']);
 self::assertSame('Hello world!', $myEntity->salute());
 ```
 
+### Test Checks
+
+To run test checks over your test suite, you can for example create umbrella test that allows you to configure and run desired checks.
```

Comment:
Tyhle dvě věty imo nedávají smysl. Zkus přeformulovat a zjednodušit, popř. to zkusím já pak

Reply:
zkus, mi to fakt prijde clear v kontextu s example

Reply:
Jo jsem to špatně pochopil, ty tu popisuješ "test checky", ne ten samotný "umbrella test". Jo, pak to dává smysl. Stejně mi ale přijde divný to "for example" ... napadá tě jiný použítí než tenhle SuiteTest? Asi bych to přeformuloval takhle:

```
Test Checks are used to assert the properties of the tests themselves.
To use them, create a test case like in the following example, using the Test Checks you want:
```

Reply:
no jakoby ze me nenapada neznamena, ze nekoho jinyho nenapadne no :D 

```
Test Checks are used to assert that tests comply with your suite's standards (are final, extend correct TestCaseBase etc.)

To run them, eg. create a test case like in the following example:
```

?

Reply:
sure, to je za mě ok :)

Reply:
anything else?